src/code/code_800EC960.c

```diff
@@ -1225,7 +1320,7 @@ void func_800EE404(void) {
     }
 }
 
-void func_800EE57C(u8 minigameIdx) {
+void Audio_OcaMinigameStart(u8 minigameIdx) {
```

Comment:
```suggestion
void Audio_OcaMemoryMinigameStart(u8 memoryGameIdx) {
```
I could consider both the ocarina memory game and the ocarina frogs game to be minigames (there's even a comment later saying `"Ocarina Frog Minigame"`, so I think this would be clearer. A few more things would need to change though like `sOcaMinigameNoteCnts`, `sOcaMinigameEndPos`, `Audio_OcaMinigameGenNote`, `gSaveContext.ocarinaGameReward`, so perhaps that'd be done in a later audio doc pass

Reply:
I figured I would change it now to be a bit more specific, but it can be revised later if needed.